cmd/metrics-api/metrics-api.go

```diff
@@ -42,10 +44,27 @@ func main() {
 		web.HealthzRoute(router, healthHandler)
 	}
 
-	log.Printf("Starting application... going to listen on %v", config.ListenAddress)
+	log.WithFields(log.Fields{"listenAddress": config.ListenAddress}).Info("Starting application")
 
 	//start
 	if err := http.ListenAndServe(config.ListenAddress, router); err != nil {
 		panic("failed to start " + err.Error())
 	}
 }
+
+func initLogger(level, format string) {
+	logLevel, err := log.ParseLevel(level)
+
+	if err != nil {
```

Comment:
not sure about this, I think we should just panic and exit

Reply:
Maybe I should add a comment to explain. I figured if a user does something silly like `LOG_LEVEL=foo` then we just set a default level of `info`.

If you think exiting is better I can do that too.

Reply:
I think it is better to exit. I think defined clear behaviour is generally better than trying to second guess the user. Id rather they know that foo is not a level than hide it behind info 😄

Reply:
I'm happy to oblige 😃

Reply:
@maleck13 Do you think I should do the same for `LOG_FORMAT`?

Reply:
I think crapping out if the value is wrong is a better experience.
It teaches the person that it's an invalid value.
What if they set it to a value like 'silly' (because they've seen this in other languages), but then see only info level. That would be confusing

Reply:
Good points. @maleck13 @david-martin I've updated it to behave as you asked!